app/sidekiq/update_seller_refund_eligibility_job.rb

```diff
@@ -10,8 +10,10 @@ def perform(user_id)
 
     if unpaid_balance_cents > 0 && user.refunds_disabled?
       user.enable_refunds!
-    elsif unpaid_balance_cents < -10000 && !user.refunds_disabled?
+    elsif unpaid_balance_cents < User::LowBalanceFraudCheck::LOW_BALANCE_THRESHOLD && !user.refunds_disabled?
       user.disable_refunds!
     end
+
+    user.restore_risk_state_if_balance_recovered!(unpaid_balance_cents)
```

Comment:
Using the existing sidekiq job on balance updates to recover risk  state instead of a new job. 

Doing it this way because
1. Updating refund state is already handled by this job 
2. Payout recovery is immediate this way
3. It blends with existing system load wrt balance updates